eng/pipelines/common/global-build-job.yml

```
@@ -206,13 +206,13 @@ jobs:
           df -h
         displayName: Disk Usage before Build
 
-      # Install Wasm dependencies: emscripten, LLVM, NodeJS
+      # Install Wasm dependencies on Windows: emscripten, LLVM, NodeJS
     - ${{ if and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.archType, 'wasm')) }}:
-      - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-emscripten.cmd $(Build.SourcesDirectory)\wasm-tools
+      - script: pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-emscripten.ps1 $(Build.SourcesDirectory)/wasm-tools
         displayName: Install/activate emscripten
-      - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.cmd $(Build.SourcesDirectory)\wasm-tools $(Build.SourcesDirectory) ${{ parameters.buildConfig }}
+      - script: pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.ps1 $(Build.SourcesDirectory)\wasm-tools $(Build.SourcesDirectory) ${{ parameters.buildConfig }}
```

```suggestion
      - script: pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.ps1 $(Build.SourcesDirectory)/wasm-tools $(Build.SourcesDirectory) -Configs ${{ parameters.buildConfig }} -CI
```
The cmake detection and `cd $(Build.SourcesDirectory)/wasm-tools` need to be migrated from `install-llvm.cmd` to `install-llvm.ps1`.

---

The cmake detection, i.e. executing `init-vs-env.cmd` as we do, doesn't seem necessary, and puts the `CMakePath` at the end anyway so doesn't precede anything already in the path.  It is possible to do from powershell, but would involve a wrapper `cmd` to `echo` the `CMakePath` back to the powershell do you scoping of environment variables and the lack or an equivalent `call` in powershell.  Question is, do we really need it?

---

I had assumed it was needed. If that's not the case - great!

---

We do need `cmake.exe` in the path, so have added this back, which means we need a `cmd` for the windows side.